package.json

```diff
@@ -2,6 +2,7 @@
   "name": "symbol-observable",
   "version": "0.2.1",
   "description": "Symbol.observable ponyfill",
+  "typings": "index.d.ts",
```

Comment:
Hmm, I thought `typings` field required explicitly. Let me quickly try and update.